packages/analytics/amplify_analytics_pinpoint_dart/lib/src/analytics_plugin_impl.dart

```diff
@@ -196,7 +187,9 @@ class AmplifyAnalyticsPinpointDart extends AnalyticsPluginInterface {
       }),
     );
 
-    _eventStorageAdapter = EventStorageAdapter(driftQueryExecutor);
+    final driftStoragePath = await _pathProvider?.getApplicationSupportPath();
+    final itemStore = DartQueuedItemStore(driftStoragePath);
```

Comment:
No need to expose the implementation details here. DartQueuedItemStore happens to use drift under the hood on VM, but if that were swapped out the underlying storage mechanism would likely still need to know what directory to put the data in.

```suggestion
    final eventStoragePath = await _pathProvider?.getApplicationSupportPath();
    final itemStore = DartQueuedItemStore(eventStoragePath);
```